apis/v1alpha1/instrumentation_webhook.go

```diff
@@ -170,6 +171,26 @@ func (r *Instrumentation) Default() {
 	if r.Spec.ApacheHttpd.ConfigPath == "" {
 		r.Spec.ApacheHttpd.ConfigPath = "/usr/local/apache2/conf"
 	}
+	if r.Spec.Nginx.Image == "" {
+		if val, ok := r.Annotations[AnnotationDefaultAutoInstrumentationNginx]; ok {
+			r.Spec.Nginx.Image = val
+		}
+	}
+	if r.Spec.Nginx.Resources.Limits == nil {
+		r.Spec.Nginx.Resources.Limits = corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("500m"),
```

Comment:
we should extract these to a constant and reuse for others.

Reply:
done